boring/src/ssl/connector.rs

```diff
@@ -189,6 +189,10 @@ impl ConnectConfiguration {
         self.verify_hostname = verify_hostname;
     }
 
+    pub fn ssl_mut(&mut self) -> &mut SslRef {
```

Comment:
Not sure I like exposing this, but there is no other way to set `TASK_CONTEXT_INDEX` from `tokio_boring::connect` prior to calling `ConnectConfiguration::connect`.

Reply:
What are your concerns?

Reply:
I managed to avoid needing this method.